lib/v2/nocturnalscans/chapters.js

```diff
@@ -0,0 +1,55 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+
+module.exports = async (ctx) => {
+    const {name} = ctx.params;
+    const url = `https://nocturnalscans.com/MANGA/${name}/`;
+    const response = await got({
+        method: 'get',
+        url,
+    });
+    const {data} = response;
+    const $ = cheerio.load(data); // 使用 cheerio 加载返回的 HTML
+    const jsPath = '#chapterlist > ul > li > div > div > a';
+    const episodes = $(jsPath).map((index, item) => {
+        const {href} = item.attribs;
+        const titlePath = `#chapterlist > ul > li:nth-child(${index + 1}) > div > div > a > span.chapternum`;
+        const title = $(titlePath).text();
+        const date = new Date(Date.now() - index * 1000).toUTCString();
+        return {
+            title, description: ``, link: href,
```

Comment:
```suggestion
            title, description: '', link: href,
```
Any reasons to use template literals for empty string?